pennylane/ops/op_math/controlled.py

```diff
@@ -127,11 +127,11 @@ def cond_fn():
         available_eps = compiler.AvailableCompilers.names_entrypoints
         ops_loader = available_eps[active_jit]["ops"].load()
         return ops_loader.ctrl(op, control, control_values=control_values, work_wires=work_wires)
-    return _ctrl(op, control, control_values=control_values, work_wires=work_wires)
+    return ctrl_python(op, control, control_values=control_values, work_wires=work_wires)
 
 
-def _ctrl(op, control, control_values=None, work_wires=None):
-    """Default `qml.ctrl` implementation"""
+def ctrl_python(op, control, control_values=None, work_wires=None):
```

Comment:
The naming feels a big strange, calling it `ctrl_python` when it is clearly written in python. Maybe call it something like `create_controlled_op`?

Reply:
```suggestion
def create_controlled_op(op, control, control_values=None, work_wires=None):
```